dashboard/src/types/tree/Tree.tsx

```diff
@@ -77,4 +77,4 @@ const origins = [
 ] as const;
 
 export const zOriginEnum = z.enum(origins);
-export const zOrigin = zOriginEnum.catch(origins[0]);
+export const zOrigin = zOriginEnum.catch(origins[2]);
```

Comment:
it would be better to use the literal `maestro` here, because if we add more items to the list the index will be affected, another option is to create a `const DEFAULT_ORIGIN = 'maestro'`

Reply:
yeah, typescript will typeguard to have only the right strings to work, so you don't need to worry typing an origin that doesn't exist here